olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -153,9 +153,9 @@ spec:
     # Further Information
     For documentation on OpenShift Serverless, see:
     - [Installation
-    Guide](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/serverless/administration-guide#installing-openshift-serverless)
+    Guide](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/serverless/install)
     - [Getting
-    started](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/serverless/serverless-getting-started)
+    started](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/serverless/develop)
```

Comment:
We should change the title as well to match the new link then, 

"Develop Serverless Applications" maybe?

Reply:
Yeah wfm.